components/input/index.ts

```diff
@@ -1,8 +1,10 @@
-import Input from './input'
+import Input, { useInputHandle } from './input'
 import Textarea from '../textarea'
 import InputPassword from './password'
 
 Input.Textarea = Textarea
 Input.Password = InputPassword
+Input.useInputHandle = useInputHandle
 
+export { useInputHandle }
```

Comment:
why export

Reply:
So that user can `import {useInputHandle} from 'components/input'`

Reply:
single use Input.useInputHandle as Select.Option? What are the benefits？

Reply:
less code
```javascript
import Input, { useInputHandle } from "components/input"

function () {
  const ... = useInputHandle(...)
  const ... = useInputHandle(...)

  return (
    ...
    <Input id="username" .../>
    <Input id="password" .../>
    ...
  )
}
```

```javascript
import Input from 'components'
const { useInputHandle } = Input

function () {
  const ... = useInputHandle(...)
  const ... = useInputHandle(...)

  return (
    ...
    <Input id="username" .../>
    <Input id="password" .../>
    ...
  )
}
```

```javascript
import Input from 'components'

function () {
  const ... = Input.useInputHandle(...)
  const ... = Input.useInputHandle(...)

  return (
    ...
    <Input id="username" .../>
    <Input id="password" .../>
    ...
  )
}
```

Reply:
so do you think all the useXXXHandle should provide the use method？

Reply:
done